calcom/cal.com
# Render # of seats and allow multiple attendees

## What does this PR do?


*As a part of the main seats feature*

This PR adds the ability for multiple attendees to attend the same booking. The number of seats and attendees should render when choosing a meeting time.

Loom demo
https://www.loom.com/share/75521ef54a5645de8eee9cf4975ea384

Still TODO
- Disable times on full times
- Send emails and reminders for new attendees
- Enable seats for event types that require confirmation from organizers

Fixes #2064

## Type of change




## How should this be tested?



- Go to an event type and under advanced options enable seats and choose a quantity
- Go to the booking page for that event type
    - The number of seats should appear under the available times
- Book that time slot
- Go back to the same event type booking page
    - You should see the number of seats updated
- Book the same time slot again with a different email
- - Go back to the same event type booking page
    - You should see the number of seats updated
